src/components/wishList/WishListView.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState, useEffect } from 'react';
+import AlbumGrid from '../AlbumGrid';
+import axiosInstance from '../../apis/axiosClient';
+
+const WishListView = () => {
+  const [albums, setAlbums] = useState();
```

Comment:
If the albums are not yet loaded (we're waiting on the response from the API request), then right now `albums` will have a value of `undefined`.

This will cause the following error in `AlbumGrid` component when it tries to map over what it expects to be a list of albums

![](https://github.com/Code-the-Dream-School/dd-prac-team4-front/assets/9094098/4d80add8-1efe-4aeb-8275-5849a6286872)

We can resolve this by setting the initial value of `albums` to an empty list; so when it first loads, AlbumGrid just gets an empty list, which it can try to call `.map(...)` on successfully but will just result in an empty grid:

```suggestion
  const [albums, setAlbums] = useState([]);
```

Reply:
Done